src/main/java/domain/user/Score.java

```diff
@@ -0,0 +1,55 @@
+package domain.user;
+
+import java.util.Objects;
+
+public class Score implements Comparable<Score> {
+    private static final int ZERO = 0;
+
+    private int value;
+
+    public Score(int value) {
+        validateScore(value);
+        this.value = value;
+    }
+
+    private void validateScore(int value) {
+        boolean isNegative = value < ZERO;
+        if (isNegative) {
```

Comment:
```suggestion
        if (value < ZERO) {
```
위와 같이 표현해도 충분히 이해하기 쉽다고 생각하는데, 어떻게 생각하시나요? 😃